tools/osemosys/features.yaml

```diff
@@ -0,0 +1,401 @@
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#operating-costs
+      value: y
+    piecewise:
+      source: []
+      value: n
+    subsidy/penalty:
+      source: []
+      value: n
+    unit:
+      source: []
+      value: n
+  asset__operating_characteristics:
+    iterative:
+      source: []
+      value: n
+    linear:
+      source: []
+      value: n
+    piecewise:
+      source: []
+      value: n
+    static non_dimensional:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#performance
+      value: y
+    static_dimensional:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#performance
+      value: y
+  asset__operating_constraints:
+    inter_spatial:
+      source: []
+      value: n
+    inter_temporal:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Advanced%20functionalities.html#short-term-operational-constraints-of-power-plants
+        - https://doi.org/10.1016/j.apenergy.2014.08.072
+      value: dev
+    operating:
+      source: []
+      value: n
+    total:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#activity-constraints
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#annual-activity-constraints
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#total-activity-constraints
+      value: y
+  asset__operating_representation:
+    ed:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#activity-variables
+      value: y
+    uced:
+      source: []
+      value: n
+  competition:
+    distributed_objectives:
+      source: []
+      value: n
+    portfolio:
+      source: []
+      value: n
+  edge_effects:
+    energy_balance:
+      source: []
+      value: n
+    spatial:
+      source: []
+      value: n
+    time_financial:
+      source:
+        - https://osemosys.readthedocs.io/en/latest/manual/Structure%20of%20OSeMOSYS.html#salvage-value
+      value: y
+    time_operational:
+      source: []
+      value: n
+  interface__input:
+    cli:
+      source:
+        - https://otoole.readthedocs.io/en/latest/
+      value: y
+    gui:
+      source:
+        - https://github.com/OSeMOSYS/MUIO
+      value: y
+    no_code:
+      source:
+        - https://github.com/OSeMOSYS/MUIO
+      value: y
+    scripting:
+      source:
+        - https://otoole.readthedocs.io/en/latest/
+        - https://github.com/OSeMOSYS/OSeMOSYS_GNU_MathProg
+      value: y
+    standardised:
+      source: []
+      value: n
+  interface__output:
+    gui:
+      source:
+      - https://github.com/OSeMOSYS/MUIO
+      value: y
+    logs:
+      source:
+        - https://en.wikibooks.org/wiki/GLPK/Using_GLPSOL
+      value: y
```

Comment:
Is this configurable directly in OSeMOSYS? Looking at the OSeMOSYS docs I guess yes since it is directly using `glpsol`

Reply:
Yes, because you *have* to use `glpsol` to create the LP file, and one of the options there is to log an output.  Likewise, when you send the LP file to the solver to solve, you get a log output.